cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/Cargo.toml

```diff
@@ -42,6 +42,7 @@ emulated-integration-tests-common = { path = "../../../common", default-features
 rococo-westend-system-emulated-network = { path = "../../../networks/rococo-westend-system" }
 rococo-system-emulated-network = { path = "../../../networks/rococo-system" }
 asset-hub-rococo-runtime = { path = "../../../../../runtimes/assets/asset-hub-rococo", default-features = false }
+penpal-runtime = { path = "../../../../../runtimes/testing/penpal", default-features = false }
```

Comment:
@claravanstaden I would remove this dep here, and just re-export `CustomizableAssetFromSystemAssetHub` here: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/integration-tests/emulated/chains/parachains/testing/penpal/src/lib.rs#L18-L20

Reply:
Done in https://github.com/paritytech/polkadot-sdk/pull/3570/commits/4c2ca7b0d4543e44486ec865fe3306b41dce1acf.